bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/components/progress/Stack.java

```diff
@@ -57,6 +59,7 @@ protected void onInitialize() {
         super.onInitialize();
 
         srOnly = new Label("srOnly", createLabelModel());
```

Comment:
Let's rename `srOnly` to `visuallyHidden` - to keep it consistent with the Bootstrap terminology